justfile

```diff
@@ -0,0 +1,35 @@
+@_default:
+    just --list
+
+# Build a development environment
+devenv:
+    uv sync --extra dev --refresh --upgrade
+
+# Run tests and linting
+test *args: devenv
+    tox {{args}}
```

Comment:
By following the README instructions, this threw an error for me using `python3 -m venv venv` > `source venv/bin/activate` as my virtual env:

> For developing crashstats-tools, clone the Git repository, create a virtual environment, and install crashstats-tools and dev dependencies with::
>    $ just devenv


I got a warning from `uv`:
```
warning: `VIRTUAL_ENV=/Users/bdanforth/Projects/venv` does not match the project environment path `.venv` and will be ignored
```

After [looking this error up](https://github.com/astral-sh/uv/issues/7778), I ultimately had to set `UV_PROJECT_ENVIRONMENT=$VIRTUAL_ENV`. 

How are you setting your virtual env up? Perhaps this is worth mentioning?

Reply:
Oh, whoops--I missed a `uv run` here. I'll fix that.